app/models/concerns/findings/unanswered.rb

```diff
@@ -60,7 +60,9 @@ def mark_as_unanswered_and_notify findings
         users.each do |user|
           findings_for_user = findings.select { |f| f.users.include? user }
 
-          NotifierMailer.unanswered_findings_notification(user, findings_for_user).deliver_later
+          findings_for_user.each do |finding|
+            NotifierMailer.unanswered_finding_notification(user, finding).deliver_later 
+          end
```

Comment:
Acá no me di cuenta antes, pero podríamos desaparecer lo de arriba, porque no hace falta más que _colectemos_ las observaciones, sería dejar algo así:
```suggestion
          findings.each do |finding|
            if f.users.include? user
              NotifierMailer.unanswered_finding_notification(user, finding).deliver_later
            end
          end
```
Y volamos la línea 61.

Reply:
impecable, sabes franco que en su momento lo habia dejado asi y nose porque no me funcionaba y lo volvi a dejar como estaba. Impecable.